Bugzilla/Config/GroupSecurity.pm

```diff
@@ -84,6 +84,32 @@ sub get_param_list {
             name    => 'strict_isolation',
             type    => 'b',
             default => 0
+        },
+        {
+            name    => 'always_filleable_groups',
```

Comment:
Sorry for the drive-by review, was just having a quick skim. Is `filleable` spelled right there? Seems like that might cause problems down the line?

Reply:
Probably should have a better name, yeah.

Reply:
Hey, @mohawk2 if you would be so kind to take a look at bugzilla/harmony#25?

Some stuff is happening in that repo because today someone submitted like 17 PRs \o/
("harmony" is "bugzilla 6 should be based on bmo" project.)

Reply:
Done.